selfdrive/car/gm/interface.py

```diff
@@ -61,6 +62,13 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, disa
       ret.radarOffCan = True  # no radar
       ret.pcmCruise = True
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_GM_HW_CAM
+      
+      if candidate in CC_ONLY_CAR:
```

Comment:
do we have full longitudinal control over these cars?

Reply:
No - not without a Pedal. The Pedal can do stop-and-go using regen braking.

@sshane demonstrated the ability to spam the button messages to the stock camera performing VOACC. Tonight is the first time it's been tried on the PT bus or with standard CC, and it works *perfectly*. This means button-spamming speed control using the stock CC should be possible. (The ECM doesn't appear to provide any other means of actuation, and my 2019 rejected the ECM firmware from a 2022 w/ACC)

I did some testing on the CC's accel and deccel ability, and was impressed. I observed it applying over half the regen braking to slow down from 70 to 24, and it accelerates comfortably. This would function fairly similar to the standard (non-ASCM, but also not VOACC) ACC option on the Silverado/Tahoe - driver would have to take over below 24 mph, and when harder braking is required. (All of these cars also have functional AEB)

Ages ago George mentioned that doing ACC by adjusting the CC speed is something he would consider (he hadn't decided yet) for community supported cars - if that's not something you want, I can keep it in a fork; it was such a eureka moment I had to leave it in here and see...

Reply:
Let's only support the ACC-equipped for now.

Reply:
@sshane no problem, I'll move it out. Is it something worth creating a draft PR for, or should it stay in OPGM for now?